src/features/events/models/EventDataModel.ts

```diff
@@ -24,6 +32,33 @@ export default class EventDataModel extends ModelBase {
     this._repo.updateEvent(this._orgId, this._eventId, { title });
   }
 
+  get state(): EventState {
+    const { data } = this.getData();
+    if (!data) {
+      return EventState.CANCELLED;
```

Comment:
It's not really cancelled when the data is missing. It likely just means that it's loading. I believe for call assignments and/or surveys we have an `UNKNOWN` state. Maybe you could use something like that here as well?

There is currently no support for cancelling events on the backend, so right now you will never see a cancelled state (that will need to be implemented later).